haiku-apps/aplayer/aplayer-4.0_svn.recipe

```diff
@@ -31,7 +31,7 @@ BUILD_PREREQUIRES="
 	cmd:g++$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:make
-	cmd:nasm$secondaryArchSuffix
+	cmd:nasm
```

Comment:
If someone tries to build aplayer for a 2nd arch, **`cmd:nasm$secondaryArchSuffix`** won't be available because SECOND_ARCHITECTURES is not defined in nasm's recipe. BTW, all the other recipes that need **`cmd:nasm`** do not have the $secondaryArchSuffix for it.
(I guess you have found the explation soon after you posted the question mark ;-)